src/main/java/core/basesyntax/BinaryString.java

```diff
@@ -7,6 +7,15 @@
      * а возвращает String с представлением этого числа в двоичном виде.
      */
     public String toBinaryString(int value) {
-        return null;
+        String binaryString = "";
+        int multiply = 1;
+        int binaryNumber = 0;
+        while (value > 0) {
+            binaryNumber += (value % 2) * multiply;
```

Comment:
```suggestion
            binaryNumber = (value % 2) + binaryNumber;
```
and then remove multiple)